### ISSUE TITLE
Converting unit to it's plural form

### ISSUE THREAD

**Issue by stenvdb** [mildlygeeky/craft-timediff#1]
How would you convert the unit to it's plural form so that it would work for any language?

**Comment by mildlygeeky** [comment]
Have you looked at the filter I added in version 1.1? That should accomplish what you're looking to do:

https://github.com/mildlygeeky/craft-timediff#translations

**Comment by stenvdb** [comment]
That always gives me the translated singular unit.
So for example, whether it is singular or plural, it always returns me "day" as unit. It should return "day" or "days" no?

**Comment by mildlygeeky** [comment]
Can you paste in the code and what it is returning? And please verify
you’re using 1.1?

**Comment by stenvdb** [comment]
Yes I am using the latest version. It is returning what it says in the docs:
`Outputs "past|1|day|singular" or "future|5|month|plural"`
I'm wondering why it doesn't return "months" instead of "month" when it is plural.

**Comment by chrisdubya** [comment]
Running into the same issue, any ideas?

**Comment by johanneslamers** [comment]
For me the same issue. @mildlygeeky any idea how to translate the plural form in another language?

**Comment by petervdbroek** [comment]
+1
